DGC.Types.schema.json

```diff
@@ -166,12 +166,12 @@
         "df": {
           "description": "ISO 8601 Date: Certificate Valid From",
           "type": "string",
-          "format": "date-time",
+          "format": "date-time"
```

Comment:
In our previous discussions about representation of time stamps we agreed to use seconds since epoch. Both in order to save space and to avoid interop-problems concerning parsing of different date-time formats.

To use seconds since epoch shouldn't be a violation of the semantics group's work. It's just another representation.

Reply:
ISO8601 is what is specified and none other.

Using "seconds since epoch" is a problematic approach as there are differences OSes, with Windows taking a different epoch baseline than e.g. *nix (which uses 1970-01-01).
See also e.g. https://www.powershelladmin.com/wiki/Convert_between_Windows_and_Unix_epoch_with_Python_and_Perl
So we stay with ISO8601, which is the regulation spec in any case

Reply:
OK